cli/organizationsettings.go

```diff
@@ -125,13 +125,13 @@ func (r *RootCmd) setOrganizationSettings(orgContext *OrganizationContext, setti
 
 				settingJSON, err := json.Marshal(output)
 				if err != nil {
-					return fmt.Errorf("failed to marshal organization setting %s: %w", inv.Args[0], err)
+					return xerrors.Errorf("failed to marshal organization setting %s: %w", inv.Args[0], err)
```

Comment:
nit: these changes are probably the result of an over-zealous linter? I don't think they're related to this PR.

Reply:
Yeah I don't remember touching these files, will revert this back.